### DISCUSSION TITLE
Naming and location of files to install polygonal landscape in Stellarium

### DISCUSSION THREAD

**Discussion by jerelGitHub** [Stellarium/stellarium#3377] [Q&A]
I am attempting to install a theodolite - generated polygonal skyline into Stellarium (v. 23.2). I'm using Mac OS 13.4 on a MacBook Pro). I have generated a landscape.ini file and a space-delimited text file for the skyline; I'm pretty sure those files are formatted correctly. I have installed the skyline.txt and a copy of the landscape.ini file in a zip archive. That zip file is installed in "home"/Library/Application Support/Stellarium/landscapes/DryWell/ folder.

The message from Stellarium when I do the installation is that the landscape was installed. But it does not become active and it does not show up in the list of available Landscapes. Does anyone know what I am doing wrong? It is not clear where the landscape.ini file is supposed to be, but I "assume" that Stellarium found what it needed.

Any ideas would be greatly appreciated.

Jerel

**Comment by alex-w** [comment]
It shouldn’t be archived in this place. Please unpack the data.

**Reply by jerelGitHub** [reply]
Thank you for getting back. I'm gathering that I should send the  landscape.ini and text file for the skyline; that sounds appropriate. please find attached…

[landscape.ini.txt](https://github.com/Stellarium/stellarium/files/12448494/landscape.ini.txt)
[TheodoliteRecord_B_8.25.23.txt](https://github.com/Stellarium/stellarium/files/12448498/TheodoliteRecord_B_8.25.23.txt)

**Comment by gzotti** [comment]
Without even looking at the contents: Filename is landscape.ini without the txt extension. See User Guide, section 5.1.1, footnote 3.

**Reply by jerelGitHub** [reply]
I do use landscape.ini in my library. When I dropped landscape.ini into the attach-files space it said it would not handle the .ini so I changed the extension to  .txt

**Comment by worachate001** [comment]
What I did to make it works:

1. In lanscape.ini, relocate 'name = DryWell' to [landscape] section.

2. In TheodoliteRecord_B_8.25.23.txt, delete lines that have the same azimuth (lines 31 & 47 in the original file).

3. Add this line in [landscape] section.
`polygonal_angle_rotatez=0.00001`

**Reply by jerelGitHub** [reply]
Worachate, thank you for  taking the time and effort to help. I now can find the Landscape in the menu. But for some reason I cannot see the landscape line in the Stellarium view. in the landscape.ini file (attached) I have included:

ground_color = .15 ,.45 ,.45
horizon_line_color = .75, .45, .45

and in the Landscape Menu I have toggled:
  Draw only polygon (thickness 5)
 show: ground; labels and illimination
These do not seem to have any effect and the stars don't seem to be eclipsed as they pass the landscape boundary.

It seems like there must be a missing parameter in my files.

Jerel


[landscape.ini.txt](https://github.com/Stellarium/stellarium/files/12456988/landscape.ini.txt)

**Reply by gzotti** [reply]
First, the "minute" sign in the latitude string must be a plain apostrophe, not whatever this is here. 
The main issue is the Macintosh line endings in the text file. Convert to CR+LF format.

**Reply by jerelGitHub** [reply]
Thank you for your detailed look. The "whatever" was a single quote. and with that and the CR/LF correction everything worked. One question I had was where the landscape.ini file belongs. Each landscape gets its own. That's perfect. I have three different spots I can shoot from around the house and each gets its own landscape. Even better would be if I could see all three polygons at once when plotting viewing strategy for a target, but this is great. And the help I've gotten from this group has been exceptional.
